utils_nlp/models/transformers/sequence_classification.py

```diff
@@ -105,44 +108,36 @@ def fit(
         batch_size=32,
         num_gpus=None,
         local_rank=-1,
+        verbose=True,
+        **kwargs,
```

Comment:
**kwargs here makes the available arguments implicit to the user. I would just expose all the arguments (except things like get_inputs) that `fine_tune` takes to the user via `fit`. Based on my experience so far, most of the arguments are needed to fine tune a model. We can always provide default values so that the user don't have to provide them if they don't want to control them.

Reply:
I'll expose the main ones for now. Feel free to add others if you need them.